io/disk/fiotest.py

```diff
@@ -120,6 +125,15 @@ def setUp(self):
             self.disk = self.workdir
 
         if self.disk in disk.get_disks():
+            if raid_needed:
```

Comment:
Can we have `self.dirs = self.disk` before this so that `else` needn't be repeated for `raid`/`lv`/`fstype`?

Reply:
We can not, since we are assigning `self.disk` to `self.dirs`, and that `self.disk` keeps changing every section.
Having said that, there is a plan to enhance this using a dictionary method.

Reply:
If we have it in top, we can still overwrite on each section?

Reply:
so that means we will overwrite in every section, which is same as now right ?

Reply:
only difference is we need no repeat code to do that

Reply:
Consider FS section. It gets self.disk from previous sections, and the prev sections can be either True or False.

if sraid is true, the self.disk will be '/dev/md/mdsraid'
else, self.disk will be input value of disk

if lv is true, self.disk again changes
if false, it is same from prev section

now when we come to fs, we need to know the latest self.disk to assign to self.dirs in the else part.

Was I clear ?

Reply:
@harish-24 apologies. Your suggestion does work.

I was worried about `self.create_fs(self.disk, self.dirs, fstype, fs_args, mnt_args)`, but missed `self.dirs = self.workdir` immediately above.